.circleci/config.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       # For some reason, circleci needs the postinstall to be run explicitly.
       # This may be unnecessary once ngcontainer uses nodejs 8
```

Comment:
it's rules_nodejs that needs to switch, not ngcontainer

Reply:
Done